libc/calls/close.c

```diff
@@ -49,7 +49,7 @@ static int close_impl(int fd) {
   }
 
   if (__isfdkind(fd, kFdZip)) {
-    if (!__vforked && _weaken(__zipos_close)) {
+    if (_weaken(__zipos_close)) {
```

Comment:
Actually, question here: why not `npassert(_weaken(__zipos_close))`? How are we winding up with a `kFdZip` file without `__zipos_close`?